webapp/utils/async_client.jsx

```diff
@@ -131,7 +131,6 @@ export function updateLastViewedAt(id) {
         () => {
             callTracker.updateLastViewed = 0;
             ErrorStore.clearLastError();
-            ErrorStore.emitChange();
```

Comment:
Yes. The emitChange function is called within the clearLastError function. We where double calling the function.